### PR TITLE
Yessenia Rodriguez javascript-this

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-02-15 at 5 23 01 PM

### CODE REVIEW

File: javascript-this/student.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* exported student */
+var student = {
+  firstName: 'Yessenia',
+  first: function () {
+    return this.firstName;
+  },
```
Comment (codingcodymiller) [inline javascript-this/student.js:6]: This can be removed

File: javascript-this/student.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  lastName: 'Rodriguez',
+  last: function () {
+    return this.lastName;
+  },
```
Comment (codingcodymiller) [inline javascript-this/student.js:10]: This can be removed

File: javascript-this/student.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  subject: 'JavaScript',
+  favorite: function () {
+    return this.subject;
+  },
```
Comment (codingcodymiller) [inline javascript-this/student.js:14]: This can be removed

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! A couple notes:

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**